config/database.php

```diff
@@ -110,19 +120,24 @@
     |
     | Redis is an open source, fast, and advanced key-value store that also
     | provides a richer set of commands than a typical key-value systems
-    | such as APC or Memcached. Laravel makes it easy to dig right in.
+    | such as APC or Memcached. Winter CMS makes it easy to dig right in.
     |
     */
 
     'redis' => [
 
         'client' => 'predis',
-        'cluster' => false,
```

Comment:
Is the code setup to still use this option if it was set somewhere?

Reply:
@LukeTowers nope, it was moved into the `options` array in Laravel, and we don't have any code that relies on this.